cumulusci/tasks/github/publish.py

```diff
@@ -54,13 +63,21 @@ def _init_options(self, kwargs):
 
         self.options["renames"] = self._process_renames(self.options.get("renames", []))
 
-        if self.options.get("version") in ("latest", "latest_beta"):
-            get_beta = self.options["version"] == "latest_beta"
-            self.options["version"] = str(
-                self.project_config.get_latest_version(beta=get_beta)
+        if "version" in self.options:  # pragma: no cover
```

Comment:
- First execution is with `warnings.warn`. It seems to include the file path and line num where the warning was issued from. 
- Second execution is with `self.logger.warn`.

I prefer `self.logger.warn`. @jstvz - Let me know what you think.

![logger_output](https://user-images.githubusercontent.com/6100538/129220003-cd92ece4-d254-4031-a026-2aa470324b4d.png)

Reply:
`logger.warn` is a deprecated spelling; `logger.warning` is preferred

Reply:
Thanks, will change. I verified that `logger.warning` identical output to `logger.warn`.

Reply:
1. I think you want `DeprecationWarning`, since `UserWarning` is the default.
2. We can handle both interactive and client use by setting `logging.captureWarnings(True)`.

But this is fine as is, IMO.